src/main/java/org/springframework/cloud/deployer/spi/mesos/chronos/ChronosTaskLauncher.java

```diff
@@ -244,5 +249,13 @@ private Double deduceCpus(AppDeploymentRequest request) {
 		return override != null ? Double.valueOf(override) : properties.getCpu();
 	}
 
+	private Collection<Constraint> deduceConstraints(AppDeploymentRequest request) {
+		Set<Constraint> requestSpecific = StringUtils.commaDelimitedListToSet(request.getDeploymentProperties().get("spring.cloud.deployer.chronos.constraints"))
```

Comment:
Shouldn't this property name be `spring.cloud.deployer.mesos.chronos.constraints`? 
Would be better to create a constant on the `prefix`.